packages/rich-text/package.json

```diff
@@ -49,18 +49,18 @@
     "@contentful/rich-text-plain-text-renderer": "^16.0.4",
     "@contentful/rich-text-types": "16.3.0",
     "@popperjs/core": "^2.11.5",
-    "@udecode/plate-basic-marks": "23.7.0",
-    "@udecode/plate-break": "23.7.0",
-    "@udecode/plate-common": "23.7.0",
-    "@udecode/plate-core": "23.6.0",
-    "@udecode/plate-list": "23.7.0",
-    "@udecode/plate-paragraph": "23.7.0",
-    "@udecode/plate-reset-node": "23.7.0",
-    "@udecode/plate-select": "23.7.0",
+    "@udecode/plate-basic-marks": "30.1.2",
+    "@udecode/plate-break": "30.1.2",
+    "@udecode/plate-common": "30.1.2",
+    "@udecode/plate-core": "30.1.2",
+    "@udecode/plate-list": "30.1.2",
+    "@udecode/plate-paragraph": "30.1.2",
+    "@udecode/plate-reset-node": "30.1.2",
+    "@udecode/plate-select": "30.1.2",
     "@udecode/plate-serializer-docx": "23.7.0",
```

Comment:
A lot of the pasting from word online and google doc break with this update. Probably better to handle in another ticket